tools/VscPreviewExtension/README.md

```diff
@@ -30,7 +30,7 @@ For further information and details about DocFX Flavored Markdown, please refere
 | `ctrl+k q` | `Open Dfm Preview to the side` | Preview to side |
 |  | `show Dfm Show` | Show Source |
 
-  ![PreviewToside](/tools/VscPreviewExtension/images/previewToSide.gif)
+  ![PreviewToside](https://github.com/dotnet/docfx/raw/master/tools/VscPreviewExtension/images/previewToSide.gif)
```

Comment:
if I use path related to docfx repo root path, just like what it used to be, it can not be translates to right absolute path when preview this markdown file.
if I use path related to docfx repo VscPreviewExtension root path(tools/VscPreviewExtension), like "images/previewToSide.gif", it works when preview, but doesn't work in the extension homepage.

---
In reply to: [126719451](https://github.com/dotnet/docfx/pull/1857#discussion_r126719451) [](ancestors = 126719451)